tf-modules/cumulus/variables.tf

```diff
@@ -242,6 +242,11 @@ variable "launchpad_certificate" {
   default = "launchpad.pfx"
 }
 
+variable "launchpad_passphrase" {
+  description = "passphrase of Launchpad certificate"
+  type    = string
```

Comment:
I think this should also have a default value of an empty string, otherwise it's required even if you want to use Earthdata for CMR/API authentication?